core-initrd/build-source-pkgs.sh

```diff
@@ -52,9 +68,11 @@ popd
 # Go through the different supported Ubuntu releases, creating source
 # packages for them.
 no_link=(debian go.mod go.sum cmd snapd vendor)
-for dir in */debian; do
-    rel=${dir%/debian}
-
+if [ "$#" -eq 0 ]; then
+    deb_dir=(*/debian)
+    set -- "${deb_dir[@]%/debian}"
+fi
+for rel; do
```

Comment:
wonder if this is slightly less obscure:

```suggestion
if [ "$#" != 0 ]; then
    # build directories passed in command line   
    dirs=("${@}")
else
    # build directories are all directories containing debian packaging
    deb_dirs=(*/debian)
    dirs="${deb_dirs[@]%/debian}"
fi
for rel in ${dirs[@]}; do
```

Reply:
I've added a comment now to clarify